cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -402,9 +405,33 @@ Depending on your installation, you may want to set these properties:
       <description>Enable Explore functionality</description>
     </property>
 
-  **Note:** This feature cannot be used unless the cluster has a correct version of Hive installed.
-  See :ref:`Hadoop/HBase Environment <install-hadoop-hbase>`. This feature is currently 
-  not supported on secure Hadoop clusters.
+  This feature cannot be used unless the cluster has a correct version of Hive installed.
+  See :ref:`Hadoop/HBase Environment <install-hadoop-hbase>`. If you have enabled the CDAP Explore Service on
```

Comment:
we have a problem with links in this document (the target here, "install-hadoop-hbase") because of duplicate use of the rest file. It is included in the Quick Start, and that is creating duplicate references (targets) that Sphinx can't resolve. the work-around is to replace the ref with a direct html link. I can get one in a couple of minutes.